scripts/update

```diff
@@ -1,8 +1,63 @@
-# If current pwd is scripts, go 1 up.
-if [ ${PWD##*/} == "scripts" ]; then
-    cd ..
+#! /bin/bash
+
+# update [HA_SRC_DIR [PID [LOG_FILE [PID_FILE]]]]
+# HA_SRC_DIR - The directory where the Home Assistant source lives
+# PID - The Process ID of Home Assistant if it is currently running. If this
+#       is supplied, Home Assistant will be shutdown before updating and then
+#       restarted after the update has been applied.
+# LOG_FILE - A file where the information usually printed to the screen should
```

Comment:
The LOG_FILE and PID_FILE logic doesn't really belong in an update script. I think this should be moved to a separate LAUNCH_DAEMON script.

We could have the `run_update` service callback decide if it should launch the regular update script or run LAUNCH_DAEMON with an update flag set.